wiki/Game_modifier/Half_Time/id.md

```diff
@@ -0,0 +1,57 @@
+---
+stub: true
+tags:
+  - half time
+  - mod
+  - game modifier
+  - HT
+---
+
+# Half Time (mod)
+
+![Ikon mod HT](/wiki/shared/mods/HT.png "Ikon mod Half Time (HT)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+
+## Tentang
+
+- Singkatan: HT
+- Tipe: Pengurang Kesulitan
+- Pengali Skor: 
+  - ![][osu!] ![][osu!taiko] ![][osu!catch]: 0.30x
+  - ![][osu!mania]: 0.50x
+- Tombol Shortcut Default: `E`
+- Caption: `Mode Siput.`
+- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Deskripsi
+
+*Catatan: Cara yang digunakan untuk memperlambat lagu tersebut bisa mengakibatkan lagunya kedengaran seperti robot atau agak keruh.*
+
+Mod **Half Time** adalah sebuah [game modifier](/wiki/Game_modifier) yang mengurangi kecepatan tempo (BPM) dari sebuah [beatmap](/wiki/Beatmap) ke 75% dari kecepatan awalnya, meingkatkan durasi dari lagu tersebut sebesar 33%, dan mengurangi sedikit jumlah [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty), [dan health (HP)](/wiki/Beatmapping/Health).
+
+### osu!taiko
+
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), lagu dan tempo yang sudah diperlambat menyebabkan not-not menjadi sangat bertumpuk karena AR yang telah dikurangi. Tetapi, karena cara denden dikalkulasikan pada osu!taiko, mereka sekarang memerlukan lebih banyak ketukan untuk diselesaikan dan mengakibatkan mereka untuk memberikan **jumlah nilai yang lebih tinggi daripada tanpa menggunakan mod** karena ketukan denden tidak terpengaruhi oleh [pengali skor](/wiki/Game_modifier/Score_multiplier).
```

Comment:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), lagu dan tempo yang sudah diperlambat menyebabkan not-not menjadi sangat bertumpuk karena jumlah dari approach rate (AR) telah dikurangi. Namun, dikarenakan kalkulasi denden pada osu!taiko memerlukan lebih banyak ketukan untuk diselesaikan, mengakibatkan pemberian **jumlah nilai menjadi lebih tinggi daripada tanpa menggunakan mod**, karena ketukan denden tidak dipengaruhi oleh [pengali skor](/wiki/Game_modifier/Score_multiplier).
```

Reply:
reword this, seems really strange to understand.

Reply:
kinda got confused there aswell